programs/marinade-finance/src/instructions/liq_pool/liquid_unstake.rs

```diff
@@ -36,8 +36,8 @@ pub struct LiquidUnstake<'info> {
         address = state.liq_pool.msol_leg
     )]
     pub liq_pool_msol_leg: Box<Account<'info, TokenAccount>>,
-    /// CHECK: in code
     #[account(mut)]
+    /// CHECK: deserialized in code, must be the one in State (State has_one treasury_msol_account)
```

Comment:
Why comment after `#[account]` not before?

Reply:
because the comment is associated with the type UnckeckedAccount, so to move them closer

Reply:
> because the comment is associated with the type UnckeckedAccount, so to move them closer

It is only in this place. All other places are not changed. It is inconsistent

Reply:
ok undoing